### DISCUSSION TITLE
Apollo Save Tool v1.7.4

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps3#99] [General]
### Added

* New cheat codes
  * Saint Seiya Sanctuary Battle
* New custom checksum support
  * Dragon Age: Origins
  * Dynasty Warriors: Gundam
  * LEGO Harry Potter: Years 1-4
  * LEGO Indiana Jones 2: The Adventure Continues
  * The Orange Box (Half-Life, Portal, TF2)

### Misc

* Updated Apollo patch engine v0.2.0
  * Improve patch error handling
  * New commands: `copy`, `endian_swap`, `msgbox`
  * New custom hash: `force_crc32`
  * Improve SW code types 9, A
  * Add SW code types 3, 7, B, C, D
  * Support initial value for `add/wadd/dwadd/wsub`
  * Fix `md5_xor` custom hash
* Updated network code to `libcurl` (TLS 1.2) 
* Changed background music using `libxmp`

### Fixed

* Improved Account/Console ID re-detection

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps3/releases/tag/v1.7.4'>Apollo Save Tool v1.7.4</a>.</em>

**Comment by diogoratomas** [comment]
Can we go back to the old music? That one is too annoying and the other one was good.

**Reply by bucanero** [reply]
> Can we go back to the old music? That one is too annoying and the other one was good.

The old music is a theme song from a Kingdom Hearts game. To avoid copyright issues, it has been removed and replaced with a public-domain S3M track.

Keep in mind, you can disable the background music from the `Settings` menu. Just open the menu, and disable the music.

**Reply by Masamune3210** [reply]
Believe me, this is better than having to deal with Sony BGM breathing down your neck lol

**Comment by FINK89i** [comment]
Ehy i have a problem preaticaly the unlock Is not working She Say that i unlokedit but when i go ti move It normaly Say to me that the copy Is blocked so i can only Copy the file Whit Apollo PS3 OS (4.89) *HEN*

**Reply by bucanero** [reply]
if you're unlocking the copy protection on HDD saves, you need to rebuild the database so the change is visible on the XMB and allows you to copy the save to USB. Also if you open the game, the game will re-apply the protection as soon as it saves again.

The easy solution is to use Apollo to copy the protected save to USB, and then use Apollo to remove the copy protection on the USB backup save.

**Comment by SSPauloSS** [comment]
Hi, I already created a new user without an account on PSN and downloaded the licenses, when I go to activate them in Apollo it says "error! fake account could not be assigned to xregistry.sys" I've tried everything, uninstalled, installed and nothing, the error always it persists, do you know how to solve it?

**Reply by bucanero** [reply]
probably an issue with the xRegistry file. My only suggestion would be to start from scratch, reinstall firmware (either CFW or HFW), then create the first account, and use Apollo to activate it.

If you have CFW, you can also try installing Evilnat CFW and use the "account activation" option available on Evilnat's firmware.

**Reply by SSPauloSS** [reply]
Mine is hen, so do i have to uninstall hen and install hen again?

**Reply by SSPauloSS** [reply]
or would i have to format the whole ps3

**Reply by bucanero** [reply]
> Mine is hen, so do i have to uninstall hen and install hen again?

That won't change the registry's file.

You can Format the disk or install a new spare drive if you have one available to test before formatting

**Reply by SSPauloSS** [reply]
like update it to original version?

**Reply by SSPauloSS** [reply]
> como atualizá-lo para a versão original?

how to install OFW and then HFW

**Reply by SSPauloSS** [reply]
> > como atualizá-lo para a versão original?
> 
> how to install OFW and then HFW

recreate the database through recovery mode?

**Comment by jdfr228** [comment]
Going to crosspost from PSX-Place in case you check here more often.

I unlocked some trophies and saw the percentage did not fully update- I have an 88% platinum on a game with no DLC and all trophies unlocked. I did around 5 trophies in a couple batches and let the auto database rebuild run.

Do trophies unlocked through Apollo have the DEX flag where they don't count towards percentage, or did I do something wrong by unlocking trophies for 2 games in a row between rebuilds? Maybe I just need to select another option on the menu, or maybe rebuild again?

**Reply by jdfr228** [reply]
Oh it also seemed to remove some unsynced Sly 2 trophies I had, but I guess that's par for the course when modifying trophy files, nothing I can't get again since I wasn't very far.
